tests/test_youtube_chapters_demo.py

```diff
@@ -0,0 +1,51 @@
+
+
+
+import os
+import sys
+import unittest
+import opentimelineio as otio
+
+
+class YoutubeChaptersDemoTests(unittest.TestCase):
+
+    def test_download_youtube_files(self):
+        youtube_url = "pvkTC2xIbeY"
+        video_file = "tmp/" + youtube_url + ".mp4"
+        description_file = "tmp/" + youtube_url + ".description"
```

Comment:
I usually recommend building templates using the .format() syntax, that way you can also kick templates up to the top level if you want as constants.

```suggestion
        video_file = "tmp/{url}/.mp4".format(url=youtube_url)
        description_file = "tmp/{url}.description".format(youtube_url)
```

or:

```python

# (top scope)
VIDEO_URL_TEMPLATE = "tmp/{url}/.mp4"
DESCRIPTION_URL_TEMPLATE = "tmp/{url}.description"

# etc...
```